### ISSUE TITLE
Unknown interpreted text role "menuselection"

### ISSUE THREAD

**Issue by SturmB** [vscode-restructuredtext/snooty-parser#28]
Please follow [the troubleshooting guide](https://docs.restructuredtext.net/articles/troubleshooting.html) to locate the culprit.

If troubleshooting does not reveal any useful information, provide the following information to create a new issue,

**Environment**
- [ ✔ ] Extension version: v155.0.0
- [ ✔ ] Visual Studio Code version: 1.59.0
- [ ✔ ] Operating System version: Windows 10 20H2 (build 19042.1151)
- [ ✔ ] Python version: 3.8.10 64-bit
- [ ✔ ] sphinx version: 4.1.2
- [ ✔ ] A sample of conf.py or its important fragments:

```py
project = 'New PC Setup'
copyright = '2021, Chris McGee'
author = 'Chris McGee'

release = '1.0.0'

extensions = [
]

templates_path = ['_templates']

exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']

html_theme = 'sphinx_rtd_theme'

html_static_path = ['_static']
```

**Expected behavior**

No linting error as I am using the correct syntax.

**Actual behavior**

A linting error appears:

```
Unknown interpreted text role "menuselection". snooty(DocUtilsParseError)
```

![image](https://user-images.githubusercontent.com/6216224/128937385-3d39ee0f-04a0-4a6f-8b97-e6ad25c8e8cb.png)

**Detailed Steps to reproduce the problem**

1. Install this extension.
2. Create a .rst file that uses the [menuselection](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection) role.
3. Witness the linting error above.

The compiled HTML output works just fine, but the linting error is really annoying.

**Comment by SturmB** [comment]
I'm sorry; was this even addressed?

**Comment by lextm** [comment]
A future VSCode extension will push the update to all users.

**Comment by lavaux** [comment]
The fix seems to cause breakage due to multiple entries having the same name:
```
Traceback (most recent call last):
  File "REDACTED/venv/jupyter-3.9/bin/snooty", line 5, in <module>
    from snooty.main import main
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/snooty/main.py", line 36, in <module>
    from . import __version__, language_server
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/snooty/language_server.py", line 32, in <module>
    from .page import Page
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/snooty/page.py", line 7, in <module>
    from .target_database import EmptyProjectInterface, ProjectInterface
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/snooty/target_database.py", line 11, in <module>
    from . import intersphinx, n, specparser
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/snooty/specparser.py", line 403, in <module>
    SPEC = Spec.loads(GLOBAL_SPEC_PATH.read_text(encoding="utf-8"))
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/snooty/specparser.py", line 283, in loads
    root = check_type(cls, toml.loads(data))
  File "REDACTED/venv/jupyter-3.9/lib64/python3.9/site-packages/toml/decoder.py", line 478, in loads
    raise TomlDecodeError("What? " + group +
toml.decoder.TomlDecodeError: What? download already exists?{'emphasis': {'help': 'Implement emphasis.', 'type': 'text'}, 'literal': {'help': 'Implement inline literal text.', 'type': 'text'}, 'code': {'help': 'Mark its content as code in a formal language.', 'type': 'text'}, 'math': {'help': 'Mark its content as mathematical notation (inline formula).', 'type': 'text'}, 'pep-reference': {'help': 'Create an HTTP reference to a PEP (Python Enhancement Proposal).', 'type': 'text'}, 'rfc-reference': {'help': 'Create an HTTP reference to an RFC (Internet Request for Comments).', 'type': 'text'}, 'strong': {'help': 'Implement strong emphasis.', 'type': 'text'}, 'title-reference': {'help': 'Describe the titles of books, periodicals, and other materials.', 'type': 'text'}, 'download': {'help': 'Link to files within your source tree that are not reST documents that can be viewed, but files that can be downloaded.', 'type': 'text'}, 'numref': {'help': 'Link to files within your source tree that are not reST documents that can be viewed, but files that can be downloaded.', 'type': 'text'}, 'token': {'help': 'The name of a grammar token.', 'type': 'text'}, 'keyword': {'help': 'The name of a keyword in Python.', 'type': 'text'}, 'eq': {'help': 'Same as math:numref.', 'type': 'text'}, 'dfn': {'help': 'Mark the defining instance of a term in the text.', 'type': 'text'}, 'mailheader': {'help': 'The name of an RFC 822-style mail header.', 'type': 'text'}, 'makevar': {'help': 'The name of a make variable.', 'type': 'text'}, 'manpage': {'help': 'A reference to a Unix manual page including the section.', 'type': 'text'}, 'manuselection': {'help': 'Mark a complete sequence of menu selections.', 'type': 'text'}, 'mimetype': {'help': 'The name of a MIME type, or a component of a MIME type.', 'type': 'text'}, 'newsgroup': {'help': 'The name of a Usenet newsgroup.', 'type': 'text'}, 'regexp': {'help': 'A regular expression.', 'type': 'text'}, 'pep': {'help': 'A reference to a Python Enhancement Proposal.', 'type': 'text'}, 'kbd': {'help': 'Mark a sequence of keystrokes.', 'type': 'text'}, 'highlight-red': {'help': 'Highlight text with a red background', 'type': 'text'}, 'highlight-yellow': {'help': 'Highlight text with a yellow background', 'type': 'text'}, 'highlight-green': {'help': 'Highlight text with a green background', 'type': 'text'}, 'highlight-blue': {'help': 'Highlight text with a blue background', 'type': 'text'}, 'subscript': {'help': 'Used to denote subscript text.', 'type': 'text'}, 'sub': {'help': 'Used to denote subscript text.', 'type': 'text'}, 'superscript': {'help': 'Used to denote superscript text.', 'type': 'text'}, 'sup': {'inherit': 'superscript'}, 'samp': {'help': 'Literal text where you may use curly braces to denote a "variable". For example, "samp:`print(1+{x})`".', 'type': 'text'}, 'guilabel': {'help': 'Used to specify a label or button in a GUI.', 'type': 'text'}, 'any': {'help': 'Used to find a valid target for its reference text.', 'type': 'text'}, 'doc': {'help': 'Direct link to documents.', 'type': 'text'}} (line 1000 column 1 char 25139)
```

Reverting the file to the previous version fixes the error.

**Comment by lextm** [comment]
@lavaux Fixed in 1.11.2.
